pyproject.toml

```
@@ -13,7 +13,7 @@ readme = "README.md"
 license = {text = "MIT"}
 requires-python = ">=3.7"
 dependencies = [
-    "jedi>=0.17.2,<0.19.0",
+    "jedi>=0.17.2,<0.20.0",
```

```suggestion
    "jedi>=0.18.0,<0.20.0",
```

0.17 is quite old now and probably doesn't support the latest Python versions.

---

Au contraire. If you don't need >=0.18 in your library, don't change it. Let the user environment and the dependencies some room and don't artificially constrain stuff, you don't need to. I know we had similar conversations in the past, and I respect your differing opinion, but I don't see any need to change this here.

---

Ok, no problem with that approach either.